lib/model_sorter.rb

```diff
@@ -0,0 +1,25 @@
+class ModelSorter
+
+  def self.sort(scope, params)
+    sort_attribute = params[:sort_attribute]
+    sort_direction = params[:sort_direction]
+
+    if sort_attribute.present? && sort_direction.present?
+      sort_scope(scope, sort_attribute, sort_direction)
+    else
+      scope
+    end
+  end
+
+private
+
+  def self.sort_scope(scope, attribute, direction)
+    case direction
+    when "asc", "desc"
+      scope.order("#{attribute} #{direction}")
```

Comment:
I believe there's a SQL injection vulnerability in here, check this out: http://rails-sqli.org/#order

A workaround would be sanitising the attribute by checking if that really belongs to one of those model attrs.